config.js.example

```diff
@@ -6,14 +6,14 @@ global.DEFAULT_LOG_LOCATION = '.';
 global.DEFAULT_RESULT_LOCATION = '/var/www/';
 global.DEFAULT_PORT = 3000;
 global.RESOURCES_WHITELIST = './whitelist.txt';
-global.TOKEN_ENDPOINT = 'https://www.w3.org/Web/publications/authorize';
+global.TOKEN_ENDPOINT = 'http://localhost:3001/authorize';
 global.SPEC_GENERATOR = 'http://localhost:3002/';
-global.USERNAME = process.env.USERNAME;
-global.PASSWORD = process.env.PASSWORD;
-global.W3C_PUBSYSTEM_URL = '';
-global.TR_INSTALL_CMD = '/bin/cp -R';
-global.UPDATE_TR_SHORTLINK_CMD = '';
-global.SENDMAIL = 'sendmail.sh';
+global.USERNAME = '';
+global.PASSWORD = '';
+global.W3C_PUBSYSTEM_URL = 'http://localhost:3001/publish';
+global.TR_INSTALL_CMD = 'mkdir -p $dest && cp -R $source $dest';
+global.UPDATE_TR_SHORTLINK_CMD = '#';
+global.SENDMAIL = './sendmail.sh';
```

Comment:
Ah. Good catch. I just checked in production to see how this could possibly work&nbsp;&mdash;&nbsp;when we deploy there, we edit it to have the absolute path to the script.